stylecheck/lint.go

```diff
@@ -375,11 +375,20 @@ func CheckErrorStrings(pass *analysis.Pass) (interface{}, error) {
 				if !ok {
 					continue
 				}
-				if !irutil.IsCallToAny(call.Common(), "errors.New", "fmt.Errorf") {
+				if !irutil.IsCallToAny(call.Common(), config.For(pass).ErrorFunctions...) {
 					continue
 				}
 
-				k, ok := call.Common().Args[0].(*ir.Const)
+				var arg ir.Value
+				// If it is a call to a method of a structure,
+				// the first argument is receiver
+				if _, ok := call.Common().Args[0].(*ir.Load); ok {
+					arg = call.Common().Args[1]
+				} else {
+					arg = call.Common().Args[0]
```

Comment:
Should we let users configure which argument contains the error string, instead of assuming that it is the first argument?

Reply:
It's not about function arguments, it's about calling a function or calling a method.

Reply:
My point is that not every function that creates errors takes the error string as the first argument. Thus, users might not only want to add function names, but also specify the index of the error string.